### PR TITLE
Adding trial end language

### PR DESCRIPTION
**Pull request by jcontonio**
Adds language for trial ending

### CODE REVIEW

File: web/src/client/js/components/Admin/AdminNotices/AdminNoticesComponent.js [hunk 1]
```diff
@@ -27,12 +27,16 @@ const AdminNoticesComponent = ({ logoutAction, organization, subscription }) =>
...
+  const trialEnds = subscription.trialEnd
+  const trialEndsInDays = moment.unix(trialEnds).diff(moment(), 'days')
 
   return (
     <div className="admin-notices">
       {TRIALING_STATUSES.includes(organization.subscriptionStatus) &&
       <div className="admin-notices__notice">
-        <h2 className="admin-notices__notice-title"><InfoIcon width="22" height="22" /> Trial period</h2>
+        <h2 className="admin-notices__notice-title">
+          <InfoIcon width="22" height="22" /> Trial {trialEnds && <>ends in {trialEndsInDays} days, {moment(trialEnds).format('MMMM Do')}</>}
```
Comment (dheniges) [inline web/src/client/js/components/Admin/AdminNotices/AdminNoticesComponent.js]: I would add "trial ends in X days, **on** [date]"

(but don't bold the on, just adding "on" to that sentence)

**Review by dheniges — CHANGES_REQUESTED** [review]
The date calculation isn't correct on this one:
Screen Shot 2019-12-19 at 4 06 00 PM

**Comment by jcontonio** [comment]
I was using the wrong Moment method, it will now do "in 10 days" or "in a few seconds", etc, using the fromNow() method.

https://momentjs.com/docs/#/displaying/fromnow/

**Review by dheniges — APPROVED** [review]
:+1:

**Pull request merged**